src/sentry/static/sentry/app/components/dropdownLink.jsx

```diff
@@ -37,10 +37,15 @@ const DropdownLink = React.createClass({
         this.props.onOpen && this.props.onOpen(e);
       }).on(
       'hidden.bs.dropdown', (e) => {
-        this.setState({
-          isOpen: false,
-        });
-        this.props.onClose && this.props.onClose(e);
+        setTimeout(() => {
+          if (!this.isMounted()) {
+            return;
+          }
+          this.setState({
+            isOpen: false,
+          });
+          this.props.onClose && this.props.onClose(e);
+        }, 0);
```

Comment:
You can omit `0`:

> delay is the number of milliseconds (thousandths of a second) that the function call should be delayed by. If omitted, it defaults to 0.

https://developer.mozilla.org/en-US/docs/Web/API/WindowTimers/setTimeout